commons/src/main/java/de/tinf15b4/kino/data/favorites/Favorite.java

```diff
@@ -16,9 +16,14 @@
         Favorite.FieldInfos.CINEMA }))
 public class Favorite extends EntityModel implements ImageContainer {
 
-    public interface FieldInfos {
-        String USER = "user";
-        String CINEMA = "cinema";
+    static class FieldInfos {
+        private FieldInfos() {
+            // Just used to hide the public constructor
+            // Sonarqube seems to like that
+        }
```

Comment:
We could easily just mark the errors as won't fix. I just did it becuase it was easy